suite-native/message-system/src/components/FeatureMessageScreen.tsx

```diff
@@ -0,0 +1,46 @@
+import { useDispatch } from 'react-redux';
+
+import { A, G } from '@mobily/ts-belt';
+
+import { Message } from '@suite-common/suite-types';
+import { messageSystemActions } from '@suite-common/message-system';
+import { useTranslate } from '@suite-native/intl';
+
+import { MessageScreen } from './MessageScreen';
+
+type FeatureMessageScreenProps = {
+    message: Message;
+};
+
+export const FeatureMessageScreen = ({ message }: FeatureMessageScreenProps) => {
```

Comment:
Please avoid passing whole objects like this. Preferably, pass `message.id` as prop and use a selector here to select the correct message.